django_celery_beat/models.py

```diff
@@ -174,6 +167,7 @@ class ClockedSchedule(models.Model):
     clocked_time = models.DateTimeField(
         verbose_name=_('Clock Time'),
         help_text=_('Run the task at clocked time'),
+        unique=True,
```

Comment:
Why limit it like this?  Having two jobs run at the same time is completely valid.

Reply:
Please see https://github.com/celery/django-celery-beat/pull/269#discussion_r294082204